Day-4/src/App.js

```diff
@@ -0,0 +1,17 @@
+import React, { createElement } from "react";
+import ReactDOM from "react-dom/client";
+import Cards from "./components/Cards";
+import Header from "./components/Header";
+import { TeamList } from "./data/data";
+
+const App = () => {
+  return (
```

Comment:
Can remove the return keyword if you don't use curly braces.

Reply:
👌